pyproject.toml

```diff
@@ -8,7 +8,8 @@ authors = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.9"
+python = ">=3.9,<3.12"
+PySide6 = "^6.4.0"
```

Comment:
FYI When I ran `poetry install` it complained that the lock file was out of sync, so I think you may need to run `poetry lock --no-update`